### ISSUE TITLE
Cannot scale backend replica count up

### ISSUE THREAD

**Issue by bryonbaker** [skupperproject/skupper-example-hello-world#8]
What is the issue?:
Cannot increase replica count once skupper expose has been executed on East. backend pod goes into crash loop backoff because it is trying to bind to the backend service.

Or more specifically, you cannot create a pod once a service has been created. This is an issue for both the front end and back end apps.

What do you expect?
To be able to increase the replica count and the pods are load balanced against frontend requests.

How to reproduce:
1. Scale backend's deployment (or frontend)  replica count to zero
2. Scale replica count to one or more.

Notes:
The last commit to the hello world backend app in main.py app added the three lines below. These break the example when you try and start a backend pod and the service has already been exposed. E.g. scale to zero and scale up to 1.

Given it is a backend I am not sure why there is a need to bind to the service. Is this a mistake?

    host = os.environ.get("BACKEND_SERVICE_HOST", "0.0.0.0")
    port = int(os.environ.get("BACKEND_SERVICE_PORT", 8080))
    uvicorn.run(star, host=host, port=port)

Log output:
```
INFO: Started server process [1]
INFO: Waiting for application startup.
INFO: Application startup complete.
ERROR: [Errno 99] error while attempting to bind on address ('172.21.208.28', 8080): cannot assign requested address
INFO: Waiting for application shutdown.
INFO: Application shutdown complete.
```

**Comment by bryonbaker** [comment]
The same problem exists for frontend. You cannot start the pods after the service has been created.

**Comment by ssorj** [comment]
Hi, Bryon.  Thank you for the bug report.  I believe I've fixed it in commit 1467396.  It was, as you suggest, a collision with the standard Kubernetes environment variables for services.
